proof_of_concept/local_workflow_runner.py

```diff
@@ -174,6 +179,13 @@ def _get_step_inputs(
 
         return step_input_data
 
+    def _retrieve_compute_asset(self, compute_asset_name: str) -> ComputeAsset:
+        # TODO: the store id is hardcoded so scenario pii test works,
+        #  but should be retrieved in some way
+        data, metadata = self._ddm_client.retrieve_data(store_id='site3-store',
+                                                        name=compute_asset_name)
+        return ComputeAsset(compute_asset_name, data, metadata)
```

Comment:
We could think about storing assets in the asset_store as a list of Dict[str, Asset]  (instead of 2 separate dicts with data and metadata), that would make this less clumsy. Let's discuss!

Reply:
Let's do this!

Reply:
Implemented this, seems more logical to me now.